scripts/tap-generate-docs/src/tap_generate_docs.clj

```diff
@@ -48,16 +50,15 @@
       (convert-object-properties schema (property-json-schema-partial "properties"))
       ;; Alternate
       (let [object-properties (convert-object-properties schema (property-json-schema-partial "properties"))
-            other-properties (let [other-properties (if (string? item-type)
-                                                      [item-type]
-                                                      (filter (partial not= "object") item-type))]
-                               other-properties)
+            other-properties (if (string? item-type)
+                               '(item-type)
```

Comment:
Why not `[item-type]`?

Reply:
Since filter returns a list, it seemed appropriate to also return a list in the literal case.